### DISCUSSION TITLE
Export and import DB

### DISCUSSION THREAD

**Discussion by Jurgens92** [kimai/kimai#4144] [Questions & Answers]
Hi, I currently have kimai 1.11.1 installed on ubuntu 18.

is there a way I can export the DB and import it into the latest kimai on ubuntu 22?

the reason behind this, every time i try and update ubuntu and kimai, it all just breaks.

thanks

**Comment by kevinpapst** [comment]
Your can try to do that and then run `bin/console kimai:update` afterwards

**Reply by Jurgens92** [reply]
update ubuntu 1st then kimai?
